src/main/java/hudson/security/ProjectMatrixAuthorizationStrategy.java

```diff
@@ -83,13 +84,21 @@ public boolean hasPermission(Authentication a, Permission permission) {
     public ACL getACL(ItemGroup g) {
         if (g instanceof Item) {
             Item item = (Item) g;
-            return (SidACL)item.getACL();
+            return item.getACL();
```

Comment:
Forgotten in #8; without this I get a (different) CCE from the `permissions` test in https://github.com/jenkinsci/workflow-plugin/pull/334.